bin/inference/pycbc_inference

```diff
@@ -61,6 +61,15 @@ def read_distributions_from_config(cp, section):
             raise ValueError("Same parameter in more than one distribution.")
     return dists
 
+def convert_liststring_to_list(lstring):
+    """ Checks if an argument of the configuration file is a string of a list
+    and returns the corresponding list (of strings)
+    """
+    if lstring[0]=='[' and lstring[-1]==']':
+        lvalue = [str(lstring[1:-1].split(',')[n].strip().strip("'"))
+                      for n in range(len(lstring[1:-1].split(',')))]
+    return lvalue
+
```

Comment:
Is it maybe worth having a InferenceConfigParser, inheriting from the current workflow one, one putting this function, and the one above, and potentially a bunch of stuff below in there?